Maple2.Server.Game/PacketHandlers/InteractObjectHandler.cs

```diff
@@ -50,6 +50,8 @@ private enum Command : byte {
                     session.Send(InteractObjectPacket.Result(InteractResult.s_interact_find_new_telescope, interact));
                     if (!session.Player.Value.Unlock.InteractedObjects.Contains(interact.Object.Id)) {
                         session.ConditionUpdate(ConditionType.interact_object, codeLong: interact.Object.Id);
+                        session.ConditionUpdate(ConditionType.interact_object_rep, codeLong: interact.Object.Id);
```

Comment:
is this condition only used in telescopes? jw

Reply:
Not sure!

Reply:
okay did a quick look. Quest ID 72000580 uses this as their condition and is for a variety of interact objects, such as golden chests and binoculars. So I think this is not just solely binoculars.

Additionally, now that I'm thinking about it, I don't think we should be locking the condition update behind if the player has interacted with. It wouldn't make sense if a player has interacted with said telescope (or any object for that matter) and then receives a quest needing to interact with that telescope. It would never complete.

Reply:
yeah sounds good, i'll fix tomorrow